examples/demo/src/App.tsx

```diff
@@ -37,7 +37,7 @@ const App = () => {
             )}
             authProvider={authProvider}
             dashboard={Dashboard}
-            loginPage={Login}
+            loginPage={<Login />}
```

Comment:
This is weird

Reply:
It does work, though. I removed it so your eyes don't bleed.